tweakwcs/linearfit.py

```diff
@@ -207,16 +207,19 @@ def iter_linear_fit(xy, uv, wxy=None, wuv=None,
     """
     if fitgeom == 'general':
         linear_fit = fit_general
+        minobj = 3
     elif fitgeom == 'rscale':
         linear_fit = fit_rscale
+        minobj = 2
+    elif fitgeom == 'rshift':
+        linear_fit = fit_rshift
+        minobj = 2
     elif fitgeom == 'shift':
         linear_fit = fit_shifts
+        minobj = 1
     else:
         raise ValueError("Unsupported 'fitgeom' value: '{}'".format(fitgeom))
 
```

Comment:
Can't you keep the following logic and just add 'rshift': 2 to the following dict and eliminate the previous tests?